### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
* [Potential Denial of Service Vulnerability in Rack](https://groups.google.com/forum/#!topic/rubyonrails-security/gcUbICUmKMc)
  => Fixed by updating of 'rack' to 1.6.2

* [Arbitrary file existence disclosure in Action Pack ](https://groups.google.com/forum/#!topic/rubyonrails-security/23fiuwb1NBA)
  => Fixed by using higher 'rails' (4.2.2) and we do not use config.serve_static_assets

* [CSRF Vulnerability in jquery-ujs and jquery-rails](https://groups.google.com/forum/#!topic/rubyonrails-security/XIZPbobuwaY)
  => Fixed by updating of 'jquery-rails' to '4.0.4'

* [XSS Vulnerability in ActiveSupport::JSON.encode](https://groups.google.com/forum/#!topic/rubyonrails-security/7VlB_pck3hU)
  => Fixed by updating 'rails' to '4.2.2'

* [IP whitelist bypass in Web Console](https://groups.google.com/forum/#!topic/rubyonrails-security/lzmz9_ijUFw)
  => We do not use 'web-console' gem at all

* [Possible Denial of Service attack in Active Support](https://groups.google.com/forum/#!topic/rubyonrails-security/bahr2JLnxvk)
  => Fixed by updating 'rails' to '4.2.2'

### CODE REVIEW

**Comment by dreamfall** [comment]
:shipit:

File: Gemfile [hunk 1]
```diff
@@ -76,7 +76,7 @@ gem 'carrierwave'
 gem "jquery.fileupload-rails", github: "bitzesty/jquery.fileupload-rails"
 gem "fog"
 gem "fog-aws"
-gem "vs_rails", "~> 0.0.7"
+gem "vs_rails", github: "bitzesty/vs-rails"
```
Comment (matthewford) [inline Gemfile]: ah - this remote url might not exist anymore - might need to revert to the published version - or i can publish a new version.

Comment (RuslanHamidullin) [inline Gemfile]: yeah, I right asked about in in vigilion chat

Comment (RuslanHamidullin) [inline Gemfile]: need to bump new version in rubygems - I have no access to it

**Pull request merged**